projects/core/src/lib/layer-collections/layer-collection.service.ts

```diff
@@ -45,4 +76,186 @@ export class LayerCollectionService {
             .getSessionStream()
```

Comment:
The session stream emits on session changes? Do you want `getSessionForRequest`, which only emits once?

Reply:
Maybe look at all the other `getSessionStream()` in this file as well if this is intended.

Reply:
This seems right, but this also seems like a bug. Maybe you can take a look.